src/ansible/roles/dns/templates/etc.dnsmasq.conf.j2

```diff
@@ -16,6 +16,9 @@ cache-size=0
 {% if 'master.ipa.test' in hostvars %}
 server=/ipa.test/{{ hostvars['master.ipa.test']['ansible_facts']['default_ipv4']['address'] }}
 {% endif %}
+{% if 'master2.ipa2.test' in hostvars %}
```

Comment:
It would be better to do a loop through all ipa servers.

Reply:
Fixed.